src/main/java/address/controller/TagListController.java

```diff
@@ -21,9 +21,9 @@
     ModelManager modelManager;
 
     @FXML
-    private AnchorPane mainPane;
+    private AnchorPane tagListMainPane;
     @FXML
-    private ScrollPane tags;
+    private ScrollPane tagListTags;
```

Comment:
Right, previously was a hotfix to solve to unique fx id, as tags is too generic.